README.md

```diff
@@ -1,2 +1,15 @@
 # seed-staged-based-messaging
-Seed Stage-Based Messaging and Content Store
+Seed Stage-Based Messaging Store
+
+## Apps & Models:
+  * contentstore
+    * Schedule
+    * MessageSet
+    * BinaryContent
+    * Message
+  * subscriptions
+    * Subscription
+
+## Metrics
+##### subscriptions.total.sum
```

Comment:
I think this is more accurately named `subscription.created.sum` - in aggregate it is the total for a given bucket size, but that is at display time.